src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/auth/securityconfigurer/ApiWebSecurityConfigurerAdapter.java

```diff
@@ -64,6 +64,10 @@ protected void configure(HttpSecurity http) throws Exception {
         filter.setExceptionIfHeaderMissing(false); // exception at this point
         // would cause http 500, we want http 401
         http
+            .authorizeRequests()
+                .antMatchers("/api/**/openapi")
```

Comment:
Probably better to use exact match `/api/v1/openapi` to be completely sure. Now it matches also `/api/v1/some-other-path/openapi`. Framework seems to take care of simple attempts to mess with this `/api/v1/foobar?trick=/openapi` but who knows, maybe there could be some kind of exploit - and I guess there's no real need for the use of the wildcard?

Reply:
Using exact match now